### ISSUE TITLE
Implement database and UI support for widgets configuration

### ISSUE THREAD

**Issue by evilaliv3** [globaleaks/GlobaLeaks#1380]
Since a few, given the growth of the adopters and the maturity of the platform the main requests that we are getting are now related to the customization of the UI and of the information shown in the relevant interfaces.

E.g. we got requests for customizing:
- the receiver lists (enabling/disablin itg, showing it to the a precise user or not, adding more info, removing info)
- for the file uploader status: (for adding removing details info etc)

in the general we approciate to every of such customization putting a node variable in a magic pot of variables that is growing reaching infinite.

what would be now valuable would to start thinking to the support for widgets configuration that would be somehow simple like and is something like what i described in past in https://github.com/globaleaks/GlobaLeaks/issues/999 but that now would be easy to be implemented as i've dont it already for fields attributes where each field type can have custom variables.

this would be a requirement for example for tickets like: https://github.com/globaleaks/GlobaLeaks/issues/1379

basically for a matter of customization we would simply need:

    config_element (id, name)
    config_element_attr(id, attr_name, attr_type, value)

where elemt.nam would be general from 'node', 'context', receivers', down to specific 'post_file_upload_widget'.

this way  /node (that would be possible to recall /config) would have simply  to serve the configuration like:

```
{
  node_config: {
      widgets_enabled: [list]
  },
  widgets_configets {
      widget_config_x: {},
      widget_config_y: {},
      widget_config_z: {}
  }
}
```

**Comment by evilaliv3** [comment]
Closing as outdated and discarded.
